spec/system/volunteers/edit_spec.rb

```diff
@@ -179,4 +179,21 @@
       expect(page).not_to have_content(inactive_casa_case.case_number)
     end
   end
+
+  describe "resend invite" do
+    let(:supervisor) { create(:supervisor, casa_org: organization) }
+
+    it "allows an admin resend invitation to a volunteer" do
```

Comment:
test description says admin but test setup is supervisor- please make them match add also a second test for admin :)
(In a follow-up PR)

Reply:
I will create another PR with the changes and  the test for admin.